<?xml version="1.0"?>
<launch>
  <!-- Camera transform calculation node -->
  <node pkg="dual_camera_simulation" type="camera_calibration_node" name="camera_calibrator" output="screen">
    <param name="marker_frame" value="aruco_marker_23" />
    <param name="camera1_frame" value="camera1_link" />
    <param name="camera2_frame" value="camera2_link" />
    <param name="base_frame" value="base_link" />
    <param name="publish_rate" value="10.0" /> <!-- TF publishing rate in Hz -->
  </node>
</launch>